Declarations/Constants.ts

```diff
@@ -123,16 +124,15 @@ export const TelemetryTypeStringToQuickPulseDocumentType: {[key in Contracts.Tel
 // OpenTelemetry Span Attributes
 export const SpanAttribute = {
     // HTTP
-    HttpHost: "http.host",
-    HttpMethod: "http.method",
-    HttpPort: "http.port",
-    HttpStatusCode: "http.status_code",
-    HttpUrl: "http.url",
-    HttpUserAgent: "http.user_agent",
+    HttpMethod: conventions.SemanticAttributes.HTTP_METHOD,
+    HttpStatusCode: conventions.SemanticAttributes.HTTP_STATUS_CODE,
+    HttpUrl: conventions.SemanticAttributes.HTTP_URL,
+    HttpUserAgent: conventions.SemanticAttributes.HTTP_USER_AGENT,
```

Comment:
All HTTP related constants [reference](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-opentelemetry-exporter/src/utils/constants/span/httpAttributes.ts)